src/PanelsView.tsx

```diff
@@ -21,28 +23,50 @@ import { useSourceActor } from './sourceMachine';
 import { SpinnerWithText } from './SpinnerWithText';
 import { StatePanel } from './StatePanel';
 
-export const PanelsView = () => {
+export const PanelsView = (props: BoxProps) => {
+  const embed = useEmbed();
   const simService = useSimulation();
   const authService = useAuth();
   const [sourceState, sendToSourceService] = useSourceActor(authService);
 
   return (
-    <ResizableBox gridArea="panels" minHeight={0}>
+    <ResizableBox
+      {...props}
+      gridArea="panels"
+      minHeight={0}
+      hidden={embed.isEmbedded && embed.mode === 'viz'}
+    >
       <Tabs
         bg="gray.800"
         display="grid"
         gridTemplateRows="3rem 1fr"
         height="100%"
+        defaultIndex={embed.panelIndex}
       >
         <TabList>
           <Tab>Code</Tab>
           <Tab>State</Tab>
           <Tab>Events</Tab>
           <Tab>Actors</Tab>
-          <Tab marginLeft="auto" marginRight="2">
+          <Tab marginLeft="auto" marginRight="2" hidden={embed.isEmbedded}>
             <SettingsIcon />
           </Tab>
-          <Login />
+          <Login hidden={embed.isEmbedded} />
+          {embed.isEmbedded && embed.showOriginalLink && embed.embedUrl && (
```

Comment:
`showOriginalLink` and `embedUrl` look a little bit confusing together here, should the latter be `originalUrl` (and the former could become `showOriginalUrl`?